clinvar-variant-types/README.md

```diff
@@ -45,37 +32,75 @@ Graphs can be enlarged by clicking on them. Dates in parentheses specify when th
   - **CompoundHeterozygote.** Presumably this should include exactly two variants which are _trans_ phased and interpreted together.
   - **Diplotype.** Similar, but at least one of the _trans_ phased alleles includes a haplotype. An example of this would be three variants located on one copy of the gene, and one variant in the second one, all interpreted together.
 
-As of July 2020, the most common case is the MeasureSet/Variant one, accounting for 1114689 out of 1117817 RCV records, or >99.7%. **Currently, this is the only type being processed by this pipeline.**
+As of 2021-03-19, the most common case is MeasureSet/Variant, accounting for 1,193,055 out of 1,196,231 RCV records, or >99.7%. **Currently, this is the only type being processed by this pipeline.** All following diagrams also examine distributions only within that record type.
+
+
+
+## Trait representation
+
+![](diagrams/traits.png)
+
+The diagram above demonstrates two properties for each Variant record:
+* Whether it contains no trait records, one record, or multiple;
+* Whether at least one of the trait records contains more than one name.
+
+Only traits of type “Disease” are processed by the pipeline and demonstrated in this diagram.
```

Comment:
It might be worth adding a diagram of the different trait types as well.

Reply:
You're right, that's an important metric. Thank you for this suggestion, I've now implemented it, and the diagram really looks much more informative. Here's an updated section in the report: https://github.com/EBIvariation/eva-opentargets/tree/eva-2352-clinvar-investigations/clinvar-variant-types#trait-representation